pkg/reconciler/pipelinerun/resources/apply_test.go

```diff
@@ -1062,6 +1082,11 @@ func TestApplyTaskRunContext(t *testing.T) {
 				Name:  "task3",
 				Value: *v1beta1.NewArrayOrString("$(tasks.task3.status)"),
 			}},
+			WhenExpressions: v1beta1.WhenExpressions{{
+				Input:    "$(tasks.task1.status)",
+				Operator: selection.In,
+				Values:   []string{"$(tasks.task3.status)"},
```

Comment:
can we add coverage for results replacements in when expressions via the tests in this file as well? (i only see status and param here, and only see results replacement tested in reconciler, end to end and examples)

Reply:
result replacements in when expressions are already tested in this file at: https://github.com/tektoncd/pipeline/blob/33c3a6972691924aa011d15bc7307b5a19775bcb/pkg/reconciler/pipelinerun/resources/apply_test.go#L291-L447